templates/3_Pixie_Cluster_Cells.ipynb

```diff
@@ -42,20 +42,46 @@
     "from ark.utils import data_utils, example_dataset, plot_utils\n",
```

Comment:
<p>Same as above, I don't think the note is needed and starting from "the path to all..." is repeated twice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>